x/tss/keeper/keeperKeygen.go

```diff
@@ -78,34 +71,52 @@ func (k Keeper) StartKeygen(ctx sdk.Context, voter types.Voter, keyInfo types.Ke
 	return nil
 }
 
-// GetKey returns the key for a given ID, if it exists
-func (k Keeper) GetKey(ctx sdk.Context, keyID exported.KeyID) (exported.Key, bool) {
-	var key exported.Key
-	ok := k.getStore(ctx).Get(pkPrefix.AppendStr(string(keyID)), &key)
-	if !ok {
-		return exported.Key{}, false
-	}
+func (k Keeper) getKeys(ctx sdk.Context) (keys []exported.Key) {
+	iter := k.getStore(ctx).Iterator(keyPrefix)
+	defer utils.CloseLogError(iter, k.Logger(ctx))
 
-	keyInfo, ok := k.getKeyInfo(ctx, keyID)
-	if !ok {
-		return exported.Key{}, false
+	for ; iter.Valid(); iter.Next() {
+		var key exported.Key
+		iter.UnmarshalValue(&key)
+
+		keys = append(keys, key)
 	}
 
-	key.Role = keyInfo.KeyRole
-	key.Type = keyInfo.KeyType
-	key.RotatedAt = k.getRotatedAt(ctx, keyID)
+	return keys
+}
 
-	return key, ok
+func (k Keeper) setKey(ctx sdk.Context, key exported.Key) {
+	k.getStore(ctx).Set(keyPrefix.AppendStr(string(key.ID)), &key)
 }
 
 // SetKey stores the given public key under the given key ID
 func (k Keeper) SetKey(ctx sdk.Context, key exported.Key) {
-	k.getStore(ctx).Set(pkPrefix.AppendStr(string(key.ID)), &key)
+	keyInfo, ok := k.getKeyInfo(ctx, key.ID)
+	if ok {
+		key.Role = keyInfo.KeyRole
+		key.Type = keyInfo.KeyType
+	}
```

Comment:
It means if `keyInfo` isn't there in the keeper, which will happen in the new chain since `keyInfo` is treated as temporary information and won't be carry over to the new chain while genesis exporting.